main.tf

```diff
@@ -311,3 +311,23 @@ resource "aws_ecr_repository_policy" "name" {
   repository = aws_ecr_repository.name[each.value].name
   policy     = join("", data.aws_iam_policy_document.resource[*].json)
 }
+
+resource "aws_ecr_registry_scanning_configuration" "test" {
```

Comment:
this should be replaced with a call to the module correct?

Reply:
Thanks yes!

Reply:
Err, rather, this needs to be deleted. Because the module is global (affects all regions), it should be deployed separately. I believe after this I'll make a new component for it.